ipatests/test_integration/test_installation.py

```diff
@@ -1836,6 +1836,45 @@ def test_replica_install_against_server(self):
                                            stdin_text=dirman_password)
         assert self.replicas[0].hostname not in cmd.stdout_text
 
+    def test_replica_install_existing_agreement(self):
+        """When the replication agreement already exists, the installer
+        should print an error message with the information on where
+        and how to remove it"""
+        # master and replica_1 already installed by class' install() method
+        tasks.stop_ipa_server(self.master)
+
+        # intentionally using ipa-server-install --unistall
+        # instead of replica-manage
```

Comment:
Do you mean `instead of replica-manage` or rather `without following the documented procedure that requires calling ipa server-del`?

Reply:
I meant `replica-manage` based on the [man page](https://linux.die.net/man/1/ipa-replica-manage) and the implementation of `uninstall_replica` method, which also uses `ipa-replica-manage`